_template/mapper-sdk/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:3.14
+RUN mkdir -p kubeedge
+COPY ./bin/Template kubeedge/bin/
```

Comment:
Should include `make` in all dockerfile, users can make the image from code directly

Reply:
Thank you for your valuable advice, I am trying to correct other mistakes : ).  
For this Dockerfile,the corresponding operation has been given in [README](https://github.com/QiQi-OvO/mappers-go/blob/main/_template/mapper-sdk/README.md), ```make mapper Template package``` can build the code and copy the compiled file to the image. Mapper and mapper-sdk-go both follow this rule, so I think it is reasonable to follow the previous ***make rules*** and ***Dockerfile***.